cli/pcluster/easyconfig.py

```diff
@@ -161,22 +159,6 @@ def configure(args):  # noqa: C901 FIXME!!!
         "Cluster Template",
         config.get("global", "cluster_template") if config.has_option("global", "cluster_template") else "default",
     )
-    aws_access_key_id = prompt(
-        "AWS Access Key ID",
-        config.get("aws", "aws_access_key_id") if config.has_option("aws", "aws_access_key_id") else None,
-        True,
-    )
-    aws_secret_access_key = prompt(
-        "AWS Secret Access Key ID",
-        config.get("aws", "aws_secret_access_key") if config.has_option("aws", "aws_secret_access_key") else None,
-        True,
-    )
-    if not aws_access_key_id or not aws_secret_access_key:
-        print(
-            "You chose not to configure aws credentials in parallelcluster config file.\n"
```

Comment:
I think we should verify if in the environment there are the credential variables and if not, print a message to the customers like: 
```
Please make sure you export a valid AWS_PROFILE or you have them exported in
the AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY environment variables.
```